xbmc/network/Network.h

```diff
@@ -129,11 +146,30 @@ class CNetwork
    bool IsConnected(void);
 
    // Return true if the magic packet was send
+   // TODO: current implementation uses ARP for MAC resolution.
+   //       as IPv6 has no ARP, it will provide expected result
+   //       only if target host is accessible via IPv4.
+   //       (anyhow it's use is safe regardless actual.
+   //       configuration, returns false for IPv6 only stack)
    bool WakeOnLan(const char *mac);
 
-   // Return true if host replies to ping
-   bool PingHost(unsigned long host, unsigned short port, unsigned int timeout_ms = 2000, bool readability_check = false);
-   virtual bool PingHost(unsigned long host, unsigned int timeout_ms = 2000) = 0;
+   /*!
+    \brief  Ping remote host
+    \param  IP host address, port (optional)
+    \return Return true if host replies to ping, false otherwise
+            Function is IPv6/v4 compatible
+            - If port is not specified, system cmd ping/ping6 is called
+            - If port is specified, host is contacted via sock connect()
+    */
+   bool PingHost(const std::string &ipaddr, unsigned short port, unsigned int timeout_ms = 2000, bool readability_check = false);
+
+   /*!
+    \brief  Ping remote host (compatibility wrapper, see PingHost(std::string&...))
```

Comment:
instead of see ... use
\sa PingHost
and doxygen will create a reference, it's possible to specify the full parameter list as well to reference a specific overload but it's only necessary for multiple overloads.